jplag/src/main/java/de/jplag/options/LanguageOption.java

```diff
@@ -13,6 +13,7 @@ public enum LanguageOption {
     PYTHON_3("python3", "de.jplag.python3.Language"),
     C_CPP("cpp", "de.jplag.cpp.Language"),
     C_SHARP("csharp", "de.jplag.csharp.Language"),
+    R("R", "de.jplag.R.Language"),
```

Comment:
Every other display name is in lower case. For consistency, we should do the same here.

Reply:
I see. In that case, I think it would be helpful to make the CLI language argument case-insensitive by changing `equals` to `equalsIgnoreCase` in `LanguageOption.fromDisplayName(String)`, what do you think?

Reply:
Sounds good!